Sage/carbon
# docs(select): document bounding rectangle helper usage

### Proposed behaviour

Document the `mockDOMRect` RTL test helper to make consumers aware of it.

### Current behaviour

The `mockDOMRect` RTL test helper is currently undocumented.  Customers would need to implement their own mock of `Element.prototype.getBoundingClientRect` in order to successfully unit-test the `Select` component

### Checklist


#### QA


### Testing instructions

Verify that:
- the documentation for `SimpleSelect` indicates how to use `mockDOMRect`;
- the documentation for `MultiSelect` links to the correct section in the `SimpleSelect` documentation;
- the documentation for `FilterableSelect` links to the correct section in the `SimpleSelect` documentation;
